lib/facter/powershell_version.rb

```diff
@@ -0,0 +1,12 @@
+require 'win32/registry'
```

Comment:
In short, this doesn't exist in non-Windows rubies and will cause issues being here.